projects/ngx-prx-styleguide/src/lib/paging/paging.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, Input, Output, EventEmitter, OnChanges } from '@angular/core';
+
```

Comment:
This is basically Peter Karman's code taken from metrics: https://github.com/PRX/metrics.prx.org/blob/master/src/app/shared/paging/episode-page.component.ts

If we decide to update metrics with this new style of paging, it will be a drop in replacement.

I also intend for us to use this on the Grove Dashboard.